packages/xrpc-server/src/types.ts

```diff
@@ -235,7 +290,7 @@ export class XRPCError extends Error {
     return {
       error: this.customErrorName ?? this.typeName,
       message:
-        this.type === ResponseType.InternalServerError
+        this.type === ClientResponseType.InternalServerError
```

Comment:
Since `this.type` is of type `ResponseType`, It seems more logical to compare this with `ResponseType.InternalServerError`:

```suggestion
        this.type === ResponseType.InternalServerError
```